modules/Movecraft/src/main/java/net/countercraft/movecraft/sign/RemoteSign.java

```diff
@@ -60,29 +61,28 @@ public final void onSignClick(PlayerInteractEvent event) {
             event.getPlayer().sendMessage("ERROR: Remote Sign can't remote another Remote Sign!");
             return;
         }
+
+        if(targetText.equalsIgnoreCase("")) {
+            event.getPlayer().sendMessage("ERROR: Remote Signs can't be blank!");
+            return;
+        }
+
         LinkedList<MovecraftLocation> foundLocations = new LinkedList<MovecraftLocation>();
         for (MovecraftLocation tloc : foundCraft.getHitBox()) {
             Block tb = event.getClickedBlock().getWorld().getBlockAt(tloc.getX(), tloc.getY(), tloc.getZ());
             if (!tb.getType().equals(Material.SIGN_POST) && !tb.getType().equals(Material.WALL_SIGN)) {
                 continue;
             }
             Sign ts = (Sign) tb.getState();
-            if (ChatColor.stripColor(ts.getLine(0)).equalsIgnoreCase(HEADER))
-                continue;
-            if (ChatColor.stripColor(ts.getLine(0)).equalsIgnoreCase(targetText)){
-                foundLocations.add(tloc);
-                continue;
-            }
-            if (ChatColor.stripColor(ts.getLine(1)).equalsIgnoreCase(targetText)){
-                foundLocations.add(tloc);
-                continue;
-            }
-            if (ChatColor.stripColor(ts.getLine(2)).equalsIgnoreCase(targetText)) {
-                foundLocations.add(tloc);
-                continue;
+
+            if (isEqualSign(ts, targetText)) {
+                if(isForbidden(ts)) {
+                    event.getPlayer().sendMessage("Warning: Forbidden remote sign found and skipped.");
```

Comment:
Instead of sending multiples of the same messages, output it as a list. For example the following:
```
Warning: Forbidden remote sign(s) found at the following locations with the following text:
- (100, 0,  30) : "Helm"
- (200, 30, 5 ) : "Remote Sign"
```
The space padding in the coordinates isn't absolutely necessary.
Furthermore, detecting any forbidden signs should cancel any other signs from triggering.